src/Stripe.net/Services/ApplePayDomains/ApplePayDomainService.cs

```diff
@@ -21,44 +21,53 @@ public ApplePayDomainService(string apiKey)
         {
         }
 
+        public override string ObjectName => "apple_pay_domain";
+
         public virtual ApplePayDomain Create(ApplePayDomainCreateOptions options, RequestOptions requestOptions = null)
         {
-            return this.Post($"{Urls.BaseUrl}/apple_pay/domains", requestOptions, options);
+            return this.CreateEntity(options, requestOptions);
         }
 
         public virtual Task<ApplePayDomain> CreateAsync(ApplePayDomainCreateOptions options, RequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken))
         {
-            return this.PostAsync($"{Urls.BaseUrl}/apple_pay/domains", requestOptions, cancellationToken, options);
+            return this.CreateEntityAsync(options, requestOptions, cancellationToken);
         }
 
         public virtual ApplePayDomain Delete(string domainId, RequestOptions requestOptions = null)
         {
-            return this.DeleteEntity($"{Urls.BaseUrl}/apple_pay/domains/{domainId}", requestOptions);
+            return this.DeleteEntityNew(domainId, null, requestOptions);
         }
 
         public virtual Task<ApplePayDomain> DeleteAsync(string domainId, RequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken))
         {
-            return this.DeleteEntityAsync($"{Urls.BaseUrl}/apple_pay/domains/{domainId}", requestOptions, cancellationToken);
+            return this.DeleteEntityAsyncNew(domainId, null, requestOptions, cancellationToken);
         }
 
         public virtual ApplePayDomain Get(string domainId, RequestOptions requestOptions = null)
         {
-            return this.GetEntity($"{Urls.BaseUrl}/apple_pay/domains/{domainId}", requestOptions);
+            return this.GetEntityNew(domainId, null, requestOptions);
```

Comment:
Why do we need `null` on GET requests?

Reply:
The `GetEntityNew` method is capable of adding parameters to retrieve requests, even though we don't do this in the library (mostly because the only case where this is needed is expansion, and this is handled separately via the `Expand*` properties on the service).

That said, I added an override to be able to call the method without an options class.

Reply:
But why since expansion is on the other options and no GET (non list) method takes an option?